docs/install/source.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-Make sure to have [Yarn installed](https://classic.yarnpkg.com/en/docs/install). It is also recommended to [install NVM (Node Version Manager)](https://github.com/nvm-sh/nvm) and use the LTS version (currently v18) of [NodeJS](https://nodejs.org/en/).
+Make sure to have [Yarn installed](https://classic.yarnpkg.com/en/docs/install). It is also recommended to [install NVM (Node Version Manager)](https://github.com/nvm-sh/nvm) and use the LTS version (currently v20) of [NodeJS](https://nodejs.org/en/).
```

Comment:
Just a note: v20 is not yet LTS version until 2023-10-24, see [nodejs-release-working-group](https://github.com/nodejs/Release#nodejs-release-working-group) for schedule